packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -60,9 +60,7 @@ export async function fetchSnapshot(
     snapshotDownloader: (url: string, fetchOptions: { [index: string]: any; }) => Promise<IOdspResponse<unknown>>,
 ): Promise<ISnapshotContents> {
     const path = `/trees/${versionId}`;
-    // We just want to fetch tree here. So explicitly set to not fetch blobs and deltas
-    // as by default server will send that.
-    const queryParams: ISnapshotOptions = { deltas: 0, blobs: 0 };
+    const queryParams: ISnapshotOptions = { deltas: 0, blobs: 1 };
```

Comment:
`fetchSnapshot` used to have an additional parameter `fetchFullSnapshot` which was removed by the PR that had changed this  - https://github.com/microsoft/FluidFramework/pull/11784/files.
Can you please make sure that it doesn't need to be added back.

Reply:
Yes, that was not required. It was always set to true.